samples/outlook/99-preview-apis/get-base64-encoding.yaml

```diff
@@ -0,0 +1,61 @@
+order: 4
+id: outlook-get-base64-encoding
+name: Get the Base64 encoding of a message (Message Read)
+description: Gets the Base64 encoding of a message in read mode.
+host: OUTLOOK
+api_set:
+    Mailbox: preview
+script:
+    content: |
+        $("#get-base64-encoding").click(getBase64Encoding);
+
+        function getBase64Encoding() {
+          Office.context.mailbox.item.getAsFileAsync((asyncResult) => {
+            if (asyncResult.status === Office.AsyncResultStatus.Failed) {
+              console.log(`Error encountered during processing: ${asyncResult.error.message}`);
+              return;
+            }
+
+            console.log(asyncResult.value);
+          });
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p class="ms-font-m">This sample shows how to get the Base64 encoding of a message in read mode.</p>
```

Comment:
This sample shows how to get the message exported in EML format, and encoded in Base64 (or something like that... should make sure that EML is included there)

Reply:
Made the change here, as well as in the DT PR.